homeassistant/components/climate/intent.py

```diff
@@ -22,7 +22,7 @@ class GetTemperatureIntent(intent.IntentHandler):
     """Handle GetTemperature intents."""
 
     intent_type = INTENT_GET_TEMPERATURE
-    slot_schema = {vol.Optional("area"): str, vol.Optional("name"): str}
+    slot_schema = vol.Schema({vol.Optional("area"): str, vol.Optional("name"): str})
```

Comment:
Does this need a developer blog post with notice of a breaking change since its changing the entity model? I might avoid this unless there is a good reason, and it looks like it could be handled in `effective_slot_schema` which is already a cached property anyway.

Reply:
The `slot_schema` is already labelled as `vol.Schema`, but you are right, there is no need to strictly require that from the intent developers.
I've done the change.